CommonProtos/tests/Unit/InstantiateClassesTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/*
+ * Copyright 2018, Google Inc.
```

Comment:
copyright in this file too. Also, should it be Google LLC now?

Reply:
Yes, updated